tests/transport_test.py

```diff
@@ -62,11 +62,10 @@ def test_parser_extensions():
     except ValueError:
         exception_thrown = True
 
-    assert exception_thrown
+    assert not exception_thrown
```

Comment:
This of course makes the statement on line 58 no longer valid, so it should be updated.

Reply:
Whoops. Missed that.
- [ ] Update comment.